README.md

```diff
@@ -123,11 +123,81 @@ pprint(feature_vector)
 }
 ```
 
+## Roadmap
+
+- [ ]  **Data Sources**
+    - [x]  [Redshift source](https://docs.feast.dev/reference/data-sources/redshift)
+    - [x]  [BigQuery source](https://docs.feast.dev/reference/data-sources/bigquery)
+    - [x]  [Parquet file source](https://docs.feast.dev/reference/data-sources/file)
+    - [ ]  Kafka source
+    - [ ]  HTTP source
+    - [ ]  Snowflake source
+    - [ ]  Synapse source
+
+- [ ]  **Offline Store**
+    - [x]  [Redshift](https://docs.feast.dev/reference/offline-stores/redshift)
+    - [x]  [BigQuery](https://docs.feast.dev/reference/offline-stores/bigquery)
+    - [x]  [In-memory / Pandas](https://docs.feast.dev/reference/offline-stores/file)
+    - [x]  [Custom offline store support](https://docs.feast.dev/how-to-guides/adding-a-new-offline-store)
+    - [ ]  Snowflake
+    - [ ]  Synapse
+
+- [ ]  **Online Store**
+    - [x]  [DynamoDB](https://docs.feast.dev/reference/online-stores/dynamodb)
+    - [x]  [Redis](https://docs.feast.dev/reference/online-stores/redis)
+    - [x]  [Datastore](https://docs.feast.dev/reference/online-stores/datastore)
+    - [x]  [SQLite](https://docs.feast.dev/reference/online-stores/sqlite)
+    - [x]  [Custom online store support](https://docs.feast.dev/how-to-guides/adding-support-for-a-new-online-store)
+    - [ ]  Postgres
+    - [ ]  Bigtable
+    - [ ]  Cassandra
+
+
+- [ ]  **Streaming**
+    - [ ]  [Custom streaming ingestion job support](https://docs.feast.dev/how-to-guides/creating-a-custom-provider)
+    - [ ]  Streaming ingestion on GCP
+    - [ ]  Streaming ingestion on AWS
+
+- [ ]  **Feature Engineering**
+    - [ ]  On-demand Transformations (Development in progress. See [RFC](https://docs.google.com/document/d/1lgfIw0Drc65LpaxbUu49RCeJgMew547meSJttnUqz7c/edit#))
+    - [ ]  Batch transformation (SQL)
+    - [ ]  Streaming transformation
+
+- [ ]  **Deployments**
+    - [ ]  Cloud Run
+    - [ ]  AWS Lambda
+    - [ ]  Kubernetes
+    - [ ]  KNative
+
+- [ ]  **Feature Serving**
+    - [x]  Python Client
+    - [ ]  REST Feature Server (Python) (Development in progress. See [RFC](https://docs.google.com/document/d/1iXvFhAsJ5jgAhPOpTdB3j-Wj1S9x3Ev_Wr6ZpnLzER4/edit))   
+    - [ ]  gRPC Feature Server (Java) (See [#1497](https://github.com/feast-dev/feast/issues/1497))
+    - [ ]  Java Client
+    - [ ]  Go Client    
+    - [ ]  Push API
+    - [ ]  Delete API
+    - [ ]  Feature Logging (for training)
+
+- [ ]  **Data Quality Management**
+    - [ ]  Metric production
+    - [ ]  Data profiling and validation (Great Expectations)
+    - [ ]  Training-serving skew detection
+    - [ ]  Drift detection
+    - [ ]  Alerting
+
+- [ ]  **Feature Discovery and Governance**
+    - [x]  Python SDK for creating and browsing objects
+    - [x]  CLI for browsing objects
+    - [x]  Model-centric tracking (feature services)
+    - [ ]  Feast Web UI
```

Comment:
A few thoughts on adding more details. First, we could put `(Scheduled for development in next N months)` next to the items we plan on tackling soon. Second, we could add a blurb down below telling folks that we welcome contributions and would be happy to help folks with building one of these features. Third, we could add some way for folks to let us know what they think about our prioritization (e.g. maybe some simple Google form asking them to rank their prioritization of the roadmap features, or voice their feature requests, etc.).

Reply:
> Third, we could add some way for folks to let us know what they think about our prioritization

I think our prioritization then needs to be clear. Would it consist only out of `Scheduled for development in next N months`? I think we should identify the stages

* Planned for Q1/2/3/4
* In design phase (see RFC-1234)
* In development
* Contribution welcome

Reply:
Those stages look good to me.